dora/job/server/src/main/java/alluxio/master/job/tracker/MigrateCliRunner.java

```diff
@@ -84,8 +84,23 @@ public CmdInfo runDistCp(AlluxioURI srcPath, AlluxioURI dstPath,
             submissionTime, filePath);
 
     try {
-      if (mFileSystem.getStatus(srcPath).isFolder()) {
-        createFolders(srcPath, dstPath, mFileSystem);
+      if (mFileSystem.exists(srcPath) && mFileSystem.getStatus(srcPath).isFolder()) {
+        if (!mFileSystem.exists(dstPath) || mFileSystem.getStatus(dstPath).isFolder()) {
+          createFolders(srcPath, dstPath, mFileSystem);
+        } else if (mFileSystem.exists(dstPath)) {
```

Comment:
why do we need to call `mFileSystem.exists(dstPath)` twice? this is  a RPCcall  and is expensive

Reply:
This is my oversight, I will focus on your opinion to reduce rpc calls.